wp-content/themes/rivard-report/less/defaults.less

```diff
@@ -846,12 +849,14 @@ aside.related {
     }
   }
   #main-nav.navbar .nav>li {
-      margin-right: 2em;
+      margin-right: 1.5em;
   }
 }
 @media (max-width: 1000px) {
+  // this does not prevent wrapping if you have way too items in the nav
   #main-nav.navbar .nav>li {
-      margin-right: 1.75em;
+      margin-right: 1.30em;
+      font-size: 13px; //1px smaller
```

Comment:
Made this margin-right even smaller, and decreased the font size one point to try to eke out some more space, but it still wraps below about 890px browser width.